### DISCUSSION TITLE
Coop Mode

### DISCUSSION THREAD

**Discussion by Suechtle** [voxeltycoon/issues#504] [General] [answered]
Hello there,

I have a friend.... and this friend likes the same sort of games which also includes building/strategy games like yours.

I suggest to add a coop mode so multiple people can play together as one single person.
So not a competetive multiplayer, but two or more people sharing the work of building a huge company.

There is a bunch of games that did this, but sadly this feature is pretty rare, especially in this genre. :/

Greetings

**Comment by katepolak** [comment, marked as answer]
This kind of MP is planned for sure but it will take quite a bit of time before that is worked on as there are more important thing to finish right now.

**Comment by victorleblais** [comment]
This feature is definitely planned, although not before quite a while since it's a pretty complicated thing to implement correctly. On the old roadmap, it is marked as "After First Alpha" which means it would be after the game is released on Steam.

**Comment by Suechtle** [comment]
Iam delighted to hear that you are going to implement such a thing.
Hopefully its not gonna take you as long as Ubisoft :D
